scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -12,156 +12,260 @@ output:
-We will continue working with samples from the [`SCPCP000005` project](https://scpca.alexslemonade.org/projects/SCPCP000005) we worked with in the integration notebook.
-This particular dataset contains 10 different samples that have been integrated using `fastMNN`, following the same procedure we outlined in `03-dataset_integration.Rmd`.
+Just like bulk RNA-seq, it is likely that one of the goals when performing single-cell RNA-seq will be to compare the gene expression of multiple samples to each other.
+Unlike bulk RNA-seq analysis, single-cell RNA-seq analysis allows us to identify and annotate cell types or subpopulations of cells present in each of our samples.
+This means that we can account for differences in cell type composition across samples and specifically focus on cell types or populations of interest when performing differential expression (DE) analysis.
+In this notebook, we will work with multiple samples to identify differentially expressed genes across cell types of interest using the [`DESeq2`](https://bioconductor.org/packages/release/bioc/html/DESeq2.html) package. 
 
-*Need to spend some time here introducing DE analysis and the packages we will be using* 
+We will continue working with samples from the [`SCPCP000005` project](https://scpca.alexslemonade.org/projects/SCPCP000005), an investigation of pediatric solid tumors led by the Dyer and Chen labs at St. Jude Children's Research Hospital.
+This particular dataset contains 10 different samples that have been integrated using `fastMNN`, following the same procedure we outlined in `03-dataset_integration.Rmd`.
+These 10 samples represent two different types of rhabdomyosarcoma (RMS): embryonal rhabdomyosarcoma (ERMS) and alveolar rhabdomyosarcoma (ARMS).
+These two subtypes can be distinguished by the presence of the `PAX3/PAX7-FOXO1` fusion gene present only in ARMS patients.
+Additionally, cells found in ARMS tumors tend to have an increased mutational burden and cells are in a more differentiated state in comparison to ERMS tumor cells ([Shern _et al._ 2014](https://doi.org/10.1158/2159-8290.CD-13-0639), [Stewart _et al._](https://doi.org/10.1016/j.ccell.2018.07.012)).
+RMS tumors, regardless of subtype, are made up of cells typically associated with development of skeletal muscle: mesoderm, myoblasts, and myocytes ([Sebire and Malone 2003](https://doi.org/10.1136/jcp.56.6.412)).
+The group that published the data we are going to look at in this notebook identified distinct differences between ARMS and ERMS tumors. 
+Here we will look at a subset of the samples they sequenced and walk through testing one of their hypothesis that cell types have distinct gene expression patterns in ARMS vs. ERMS samples.
 
 ## Set Up
 
-```{r}
+```{r setup}
+# set seed for reproducibility
 set.seed(2022)
 
-library(magrittr)
-library(SingleCellExperiment)
-library(ggplot2)
+# load libraries
+library(magrittr) # the pipe (%>%)
```

Comment:
dunno if you want to do the pipe substitution for this PR...

Reply:
I actually had meant to do that in the next PR and do a replace all. I'll add it in there instead.

Reply:
Just noting that this is happening in #600 instead